codeSnippets/snippets/tutorial-full-stack-task-manager/shared/src/commonMain/kotlin/Platform.kt

```diff
@@ -0,0 +1,5 @@
+interface Platform {
```

Comment:
Is this file used?

Reply:
Yes, it's also referenced in [Examine the project](https://ktor.shared.aws.intellij.net/docs/full-stack-development-with-kotlin-multiplatform.html#examine-project)